ord_schema/visualization/drawing.py

```diff
@@ -0,0 +1,80 @@
+"""This module contains two molecular drawing functions to render SVGs or PNGs
```

Comment:
Should these files be in a separate PR?

Reply:
The text generation and HTML generation uses the same base code. The drawing module is needed for HTML generation. Should I separate them?

Reply:
I think that might be good style, but I'm not going to require it to merge. You could have the text generation module output either txt or html and then have the drawing module use the html output for its own stuff; we might want to use the html in other settings as well...